code/modules/clothing/factions/hardliners.dm

```diff
@@ -0,0 +1,181 @@
+
+/obj/item/clothing/suit/toggle/armor/vest/hardliners
+	name = "hardliners captain coat"
+	desc = "An imposing armored coat worn by captains of Hardliner fleets, hand-designed by Cybersun tailors to provide maximum protection to its wearer."
+	body_parts_covered = CHEST|GROIN|ARMS
+	icon_state = "hl_captain"
+	item_state = "hl_captain"
+	icon = 'icons/obj/clothing/faction/hardliners/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/hardliners/suits.dmi'
+	blood_overlay_type = "coat"
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30, "energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	togglename = "buttons"
+
+///////////////
+//Spacesuits//
+//////////////
+
+/obj/item/clothing/head/helmet/space/hardsuit/syndi/hl
+	name = "white-red hardsuit helmet"
+	desc = "An advanced dual-mode helmet derived from ICW-era advanced special operations helmets, its red partly replaced by white. It is in EVA mode. Manufactured by Cybersun."
+	alt_desc = "An advanced dual-mode helmet derived from ICW-era advanced special operations helmets, its red partly replaced by white. It is in combat mode. Manufactured by Cybersun."
+	icon_state = "hardsuit1-hl"
+	item_state = "hardsuit1-hl"
+	icon = 'icons/obj/clothing/faction/hardliners/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/hardliners/head.dmi'
+	hardsuit_type = "hl"
+
+/obj/item/clothing/suit/space/hardsuit/syndi/hl
+	name = "white-red hardsuit"
+	desc = "An advanced dual-mode hardsuit derived from ICW-era advanced special operations hardsuits, its red partly replaced by white. It is in EVA mode. Manufactured by Cybersun."
+	alt_desc = "An advanced dual-mode hardsuit derived from ICW-era advanced special operations hardsuits, its red partly replaced by white. It is in combat mode. Manufactured by the Cybersun."
+	icon_state = "hardsuit1-hl"
+	item_state = "hardsuit1-hl"
+	hardsuit_type = "hl"
+	icon = 'icons/obj/clothing/faction/hardliners/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/hardliners/suits.dmi'
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/syndi/hl
+	lightweight = 1
+	jetpack = null
+
+/////////
+//Hats//
+////////
+
+/obj/item/clothing/head/hardliners
+	name = "white surgical cap"
+	desc = "A surgical cap used by doctors of Hardliner fleets, matching their white smocks."
+	icon_state = "hl_surgery"
+	icon = 'icons/obj/clothing/faction/hardliners/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/hardliners/head.dmi'
+
+
+/obj/item/clothing/head/hardhat/hardliners
+	name = "white-red hard hat"
+	desc = "A white-red hardhat typically used by both miners and mechanics under the Hardliner fleets."
+	icon_state = "hl_hardhat"
+	icon = 'icons/obj/clothing/faction/hardliners/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/hardliners/head.dmi'
+
+/obj/item/clothing/head/hardliners/peaked
+	name = "2nd Battlegroup peaked cap"
```

Comment:
Should this actually be 2nd Battlegroup? Could be confusing since the NGR was previously known as the 2nd Battlegroup, and the actual ICW-era 2BG uniforms probably wouldn't have had the Hardliners' signature white.